### ISSUE TITLE
"Copy raw transaction to clipboard" with pending transaction

### ISSUE THREAD

**Issue by c-geek** [duniter/sakia#469]
This feature does not work for pending transactions, that is transactions that have not been written to the blockchain yet.

**Comment by Insoleet** [comment]
Yes, Sakia does not store the raw transaction once it has been sent. It only stores the sha hash.

Do you really need such a feature ?
